src/imgui/csvparser.hpp

```diff
@@ -0,0 +1,188 @@
+#ifndef CSV_PARSER_H
+#define CSV_PARSER_H
+
+#include "fileHandler.hpp"
+#include "error.hpp"
+
+#include <string>
+#include <tl/expected.hpp>
+
+namespace CSV {
```

Comment:
please comply with the other namespaces nomenclatures

Reply:
Done|